src/main/java/vazkii/patchouli/common/handler/LecternEventHandler.java

```diff
@@ -0,0 +1,63 @@
+package vazkii.patchouli.common.handler;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.block.LecternBlock;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.entity.player.ServerPlayerEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.tileentity.LecternTileEntity;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.minecraftforge.event.entity.player.PlayerInteractEvent;
+import net.minecraftforge.eventbus.api.Event;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+
+import vazkii.patchouli.api.PatchouliAPI;
+import vazkii.patchouli.common.book.Book;
+import vazkii.patchouli.common.util.ItemStackUtil;
+
+@Mod.EventBusSubscriber(bus = Mod.EventBusSubscriber.Bus.FORGE)
+public class LecternEventHandler {
+
+	@SubscribeEvent
+	static void rightClick(PlayerInteractEvent.RightClickBlock event) {
+		World world = event.getWorld();
+		BlockPos pos = event.getPos();
+		BlockState state = world.getBlockState(pos);
+		if (!world.isRemote) {
```

Comment:
push this check down. player interact events should ideally have the same results (event.setUseBlock/setUseItem) both client and serverside, or else you could get desyncs.

Reply:
done